### PR TITLE
Updated Documentation

### PR DESCRIPTION
**Pull request by Storm-Engineering**
### Summary of Changes

* Made paths to files that we referenced a bit easier to find by using the full path
* Used javascript instead of diff marking, hopefully will be easier to read

### CODE REVIEW

File: src/actions/tests/__snapshots__/pending-breakpoints.spec.js.snap [hunk 1]
```diff
@@ -123,3 +123,28 @@ Object {
...
+    "line": 5,
+    "sourceUrl": "http://localhost:8000/examples/foo2",
+  },
+  "location": Object {
+    "column": undefined,
+    "line": 5,
+    "sourceUrl": "http://localhost:8000/examples/foo2",
+  },
+}
+`;
```
Comment (jasonLaster) [inline src/actions/tests/__snapshots__/pending-breakpoints.spec.js.snap]: how'd this change :)

Comment (Storm-Engineering) [reply src/actions/tests/__snapshots__/pending-breakpoints.spec.js.snap]: AGH!  Thank you!  I couldn't figure out why this wouldn't compile correctly, as I had only changed documentation.  I was trying to push this to my fork and it ran all the tests and it seemed to have changed this.

**Comment by jasonLaster** [comment]
if you rebase, you'll get the fix for our intermittent CI issue :/

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -255,9 +255,9 @@ debugger.html community team members help shepherd the community.
 They are here to help mentor new comers, review pull requests, and facilitate issue discussions.
 They are a fantastic resource and genuinely friendly human beings.
 
-| | | | | | |
-| ---------- | ------ | ----- | ---- | ---- | ---- |
-| ![][bomsy] <br /> [@bomsy][@bomsy] | ![][jbhoosreddy] <br /> [@jbhoosreddy][@jbhoosreddy] | ![][wldcordeiro] <br />[@wldcordeiro][@wldcordeiro] | ![][irfanhudda] <br />[@irfanhudda][@irfanhudda] | ![][darkwing] <br />[@darkwing][@darkwing] | ![][nyrosmith] <br />[@nyrosmith][@nyrosmith] |
+| | | | | | | |
+| ---------- | ------ | ----- | ---- | ---- | ---- | ---- |
+| ![][bomsy] <br /> [@bomsy][@bomsy] | ![][jbhoosreddy] <br /> [@jbhoosreddy][@jbhoosreddy] | ![][wldcordeiro] <br />[@wldcordeiro][@wldcordeiro] | ![][irfanhudda] <br />[@irfanhudda][@irfanhudda] | ![][darkwing] <br />[@darkwing][@darkwing] | ![][nyrosmith] <br />[@nyrosmith][@nyrosmith] | ![][lukaszsobek] <br />[@lukaszsobek][@lukaszsobek]
```
Comment (jasonLaster) [inline CONTRIBUTING.md]: looks like a merge conflict

File: docs/updates/README.md [hunk 3]
```diff
@@ -165,7 +165,7 @@ We focused this week on getting lots of bugfixes and community prs in
 * Nightly is up to date with the latest debugger updates
 * new styling for the Outline View, big thanks to [@amelzer]
 * searchbar had a lot of improvements
-* lots of dependancies were updated, thanks to [@zaggy]
+* lots of dependencies were updated, thanks to [@zaggy]
```
Comment (jasonLaster) [inline docs/updates/README.md]: merge conflict, maybe

File: src/actions/sources.js [hunk 4]
```diff
@@ -114,19 +114,8 @@ export function newSources(sources: Source[]) {
...
-    dispatch({
-      type: "ADD_SOURCES",
-      sources: filteredSources
-    });
-
     for (const source of filteredSources) {
-      dispatch(loadSourceMap(source));
-      dispatch(checkSelectedSource(source));
-      dispatch(checkPendingBreakpoints(source.id));
+      dispatch(newSource(source));
```
Comment (jasonLaster) [inline src/actions/sources.js]: conflict :/

File: CONTRIBUTING.md (hunk 2 above)
Comment (Storm-Engineering) [reply CONTRIBUTING.md]: Okay, did you get my message in slack?  I thought I was on the right track for rebasing the current devtools-html/master with mine and then pushing my changes to my fork.  I guess i was not...

**Comment by Storm-Engineering** [comment]
I am going to redo this pull request so it is cleaner.  This is too messy :|

**Pull request closed without merging**